java/drivers/driver-hazelcast4plus/pom.xml

```diff
@@ -71,6 +71,12 @@
             <version>${hazelcast.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-enterprise</artifactId>
```

Comment:
this causes problems with some tests that lack license. This will also make it harder for the users to compile simulator with only OS code

Reply:
ugly hack to not use EE in tests (currently not needed): https://github.com/hazelcast/hazelcast-simulator/pull/2223/commits/2378426deb37ba52bf16c28719d14fce7a98c20c

Reply:
we have https://github.com/hazelcast/hazelcast-ee-simulator-tests but it seems to be dead. @gareth-johnston maybe you know about a good place for EE tests?

Reply:
First I am learning about this repo @k-jamroz  🤔  I guess internal benchmarks is an option... Maybe we should think about reviving the dead repository.